app/Controllers/feedController.php

```diff
@@ -385,10 +385,14 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 			}
 			if ($simplePiePush === null && $feed_id === 0 && (time() <= $feed->lastUpdate() + $ttl)) {
 				//Too early to refresh from source, but check whether the feed was updated by another user
-				if ($mtime <= 0 || $feed->lastUpdate() >= $mtime) {
+				$ε = 10;	// negligible offset errors in seconds
```

Comment:
```suggestion
				$errorInSeconds = 10;	// negligible offset errors in seconds
```
it's more explicit

Reply:
It is not an error, it is a negligible offset, commonly called epsilon :-)